docs/fastdds/xml_configuration/datawriter.rst

```diff
@@ -132,12 +132,14 @@ The DataWriter configuration is performed through the XML elements listed in the
     :language: xml
     :start-after: <!-->XML-DATAWRITER<-->
     :end-before: <!--><-->
-    :lines: 2-4, 6-71, 73-74
+    :lines: 2-4, 6-81, 83-84
```

Comment:
The final lines are not the correct ones (they are also wrong in the current version but I think we can fix that in this PR)

Reply:
2ab1b34